### PR TITLE
EG-3661 Add Kotlin instructions on Running a sample CorDapp page

### PR DESCRIPTION
**Pull request by taylorharvey-r3**
Added Kotlin instructions and fixed links across CE and OS 4.4 and 4.5.

### CODE REVIEW

File: content/en/docs/corda-enterprise/4.4/cordapps/tutorial-cordapp.md [hunk 1]
```diff
@@ -48,37 +47,51 @@ This section describes how to deploy and run the example CorDapp on the followin
...
-  * Click **File** >  **Project Structure**.
-  * Under **Project Settings**, click the **Project** option (if not displayed by default).
-  * In the **Project SDK** section, click **New…** > **JDK**, then select the home directory of your JDK and click **OK**.
-* Specify the following additional settings:
-  * Select **Modules**, then click the **+** button located just above the `cordapp-example` folder and select **Import Module**.
-  * On the **Select File or Directory to Import** window, navigate to `samples-java\Basic\cordapp-example` and click **OK**.  
-  * On the **Import Module** window, select the **Import module from external model** option, then select **Gradle** from the list of options and click **Next**.
-  * Select the **Use auto-import** checkbox and click **Finish**, then click **OK**.
+1. Open IntelliJ.
+2. From the splash screen, click **Open**, navigate to the `Basic\cordapp-example` sub-folder, and click **OK**. The project containing the sample CorDapp should open.
```
Comment (benpesterr3) [inline content/en/docs/corda-enterprise/4.4/cordapps/tutorial-cordapp.md:83]: * Is this one step? 
* For step result - maybe take out 'should'. Step result would be: The sample CorDapp opens.

**Review by benpesterr3 — APPROVED** [review]
Have approved but there is a comment for you to check before you merge.

**Review by benpesterr3 — APPROVED** [review]

**Comment by taylorharvey-r3** [comment]
Looks good on docs-staging. @benpesterr3

**Comment by ivanterziev-r3** [comment]
@taylorharvey-r3 I guess this warrants a propagation fix for OS / CE 4.6 in corda-docs-develop?

**Comment by taylorharvey-r3** [comment]
@ivanterziev-r3 Mary already fixed those. She just needed me to copy over changes to OS/CE 4.4 and 4.5.

**Comment by taylorharvey-r3** [comment]
@ivanterziev-r3 Should have included this above: https://github.com/corda/corda-docs-develop/pull/472

**Pull request merged**